scripts/clean

```diff
@@ -14,8 +14,11 @@ clean_package() {
     return
   fi
 
-  # Use a wilcard here to remove all versions of the package
-  for i in "${BUILD}/${1}-"*; do
+  # Use a wildcard here to remove all versions of the package
+  for i in "${BUILD}/${1}-"* \
```

Comment:
Do we need to check `"${BUILD}/${1}-"*`? There's no need to support legacy build directories.

Reply:
The idea was to be nice to ppl doing incremental builds :) But sure, I can get rid of it